frontends/concrete-python/concrete/fhe/mlir/processors/assign_bit_widths.py

```diff
@@ -396,207 +396,285 @@ def min_max(self, node: Node, preds: List[Node]):
     # Operations
     # ==========
 
-    add = {
-        inputs_and_output_share_precision,
-    }
```

Comment:
@rudy-6-4 wanted it, he said it can hide bugs. Tell me what's the choice of the team and I apply it!

Reply:
It can in generic python code. But this class is like a Rust macro, it has it's own syntax and it's not used anywhere else other than that file. That's why I suggested to only disable it for this file instead of all files. @rudy-6-4 wdyt?

Reply:
ah yes, just in the file (or in the section) sounds like a nice idea

Reply:
removed that; was not able to have it per file, but did it per line